### PR TITLE
drop all capabilities from elastic service

### PR DESCRIPTION
**Pull request by pgvishnuram**
## Description

Dropping all capabilities from elasticsearch, this will allow enterprise customer to install elasticsearch with no extra privileges 

## Related Issues

https://github.com/astronomer/issues/issues/4226

## Testing

Yet to update

## Merging

Do not merge this PR until it lists which release branches this PR should be merged / cherry-picked into.

### CODE REVIEW

**Comment by pgvishnuram** [comment]
@danielhoherd -Do we need add an test case to validate

**Comment by pgvishnuram** [comment]
@vishwas-astro @rob-1126

**Comment by andriisoldatenko** [comment]
yeah it would be great to add tests, may be document it somehow.

**Review by rob-1126 — APPROVED** [review]

File: tests/chart_tests/test_elasticsearch.py [hunk 1]
```diff
@@ -99,3 +99,45 @@ def test_elasticsearch_securitycontext_defaults(self, kube_version):
...
+        )
+        assert len(docs) == 2
+        for doc in docs:
+            pod_data = doc["spec"]["template"]["spec"]["containers"][0]
+            assert pod_data["securityContext"]["capabilities"]["drop"] == ["ALL"]
+            assert pod_data["securityContext"]["runAsNonRoot"] is True
+            assert pod_data["securityContext"]["runAsUser"] == 1000
+
+    def test_elasticsearch_securitycontext_overrides(self, kube_version):
+        """Test  ElasticSearch master, data  with securitycontext custom values"""
```
Comment (danielhoherd) [inline tests/chart_tests/test_elasticsearch.py:121]:
```suggestion
        """Test ElasticSearch master, data with securityContext custom values"""
```

File: tests/chart_tests/test_elasticsearch.py [hunk 2]
```diff
@@ -99,3 +99,45 @@ def test_elasticsearch_securitycontext_defaults(self, kube_version):
             assert doc["spec"]["template"]["spec"]["securityContext"] == {
                 "fsGroup": 1000
             }
+
+    def test_elasticsearch_securitycontext_defaults(self, kube_version):
+        """Test  ElasticSearch master, data with securitycontext default values"""
```
Comment (danielhoherd) [inline tests/chart_tests/test_elasticsearch.py:104]:
```suggestion
        """Test ElasticSearch master, data with securityContext default values"""
```

**Review by danielhoherd — APPROVED** [review]

**Pull request merged**